canbench-bin/src/fmt.rs

```diff
@@ -1,3 +1,6 @@
+const POSITIVE_INFINITY_PERCENT: &str = "1.0E100%";
+const NEGATIVE_INFINITY_PERCENT: &str = "-1.0E100%";
```

Comment:
this looks strange to me (in particular, since the value is arbitrary); is there no alternative to +inf% interpreted as a number?

Reply:
I failed to find any good alternative, many people just use big round number.